%YAML:1.0
---
hog32: !!opencv-object-detector-hog
   winSize: [ 32, 32 ]
   blockSize: [ 16, 16 ]
   blockStride: [ 8, 8 ]
   cellSize: [ 8, 8 ]
   nbins: 22
   derivAperture: 1
   winSigma: 4.
   histogramNormType: 0
   L2HysThreshold: 2.0000000000000001e-01
   gammaCorrection: 0
   nlevels: 64
   signedGradient: 1
